apps/api/src/app/notifications/e2e/get-activity-feed.e2e.ts

```diff
@@ -143,4 +143,51 @@ describe('Get activity feed - /notifications (GET)', async () => {
     expect(activities.length).to.equal(1);
     expect(activities[0]._templateId).to.equal(template._id);
   });
+
+  it('should filter by subscriberId', async function () {
+    const subscriberIdToCreate = SubscriberRepository.createObjectId() + 'some-test';
+
+    await session.triggerEvent(
+      template.triggers[0].identifier,
+      {
+        subscriberId: subscriberIdToCreate,
+        email: 'test@email.coms',
+      },
+      {
+        firstName: 'Test',
+      }
+    );
+
+    await session.triggerEvent(template.triggers[0].identifier, SubscriberRepository.createObjectId(), {
+      firstName: 'Test',
+    });
+
+    await session.triggerEvent(
+      template.triggers[0].identifier,
+      {
+        subscriberId: SubscriberRepository.createObjectId(),
+      },
+      {
+        firstName: 'Test',
+      }
+    );
```

Comment:
Is this repeated on purpose?

Reply:
Yes, we need to run this twice, so was just copying this from previous test. So here we are testing the subscriber Id search behaviour.